lesson_05/script.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+const ul = document.getElementById("js-list");
+const fragment = document.createDocumentFragment();
+const data = [
+  { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+];
+
+const result = new Promise(function (resolve, reject) {
+  for (let i = 0; i < data.length; i++) {
+    const li = document.createElement("li");
+    const a = document.createElement("a");
+    const img = document.createElement("img");
+    a.href = data[i].to;
+    a.text = data[i].text;
+    img.alt = data[i].alt;
+    fragment
+      .appendChild(li)
+      .appendChild(a)
+      .insertAdjacentElement("afterbegin", img);
+  }
+});
+ul.appendChild(fragment);
```

Comment:
It seems that there are not enough items to specify the image.

Reply:
Thanks to the reference site, I think I have made some progress. I still have some unclear points to review.　　[`c99eb61`](https://github.com/Numabeharuka/Lesson-js/pull/6/commits/c99eb6128a95c15bbad8745f5b08b44c92dca0d4)